Code/Framework/AzTest/AzTest/Platform/Linux/AzTest_Traits_Linux.h

```diff
@@ -17,9 +17,22 @@
 #define AZ_TRAIT_DISABLE_FAILED_EMOTION_FX_TESTS true
 #define AZ_TRAIT_DISABLE_FAILED_EMOTION_FX_EDITOR_TESTS true
 
-// Golden perline gradiant values for random seed 7878 for this platform
-#define AZ_TRAIT_UNIT_TEST_PERLINE_GRADIANT_GOLDEN_VALUES_7878  0.5000f, 0.4455f, 0.4838f, 0.4589f, \
-                                                                0.4539f, 0.5138f, 0.4104f, 0.4618f, \
-                                                                0.4572f, 0.4894f, 0.4767f, 0.4197f, \
-                                                                0.5500f, 0.4635f, 0.5199f, 0.4467f, \
+
+// Golden perline gradiant values for random seed 7878 for this platform. Currently only ubuntu 20.04 and 22.04 
+// supports the gradient test for perline
+#if defined(LY_UBUNTU_VERSION)
+    #if LY_UBUNTU_VERSION == 2004
+
+        #define AZ_TRAIT_UNIT_TEST_PERLINE_GRADIANT_GOLDEN_VALUES_7878  0.5000f, 0.4455f, 0.4838f, 0.4589f, \
```

Comment:
Looks like it's the same name as before, but contains a typo of 'Perlin'

Reply:
The behavior of std::mt19937 being different even across different distros wwas a surpise to me. But responding to @lumberyard-employee-dm suggestion, mocking would be the better approach